app/Http/Controllers/Api/PermissionController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use Spatie\Permission\Models\Permission;
+
+class PermissionController extends Controller
+{
+    public function index()
```

Comment:
return type is missing in the func definition

Reply:
Added.